python/python_fastapi_server/routers/geojsonresponse.py

```diff
@@ -0,0 +1,13 @@
+import json
+from typing import Any
+
+from fastapi import Response
+from fastapi.encoders import jsonable_encoder
+
+
+class GeoJSONResponse(Response):
+    media_type = "application/geo+json"
+
+    def render(self, content: Any) -> bytes:
+        return content.json(exclude_none=True)
+
```

Comment:
This is no longer needed, use this instead:
```suggestion
class GeoJsonResponse(JSONResponse):
    media_type = ResponseFormatOptions.geojson.value
```